OracleHTTPServer/README.md

```diff
@@ -1,82 +1,47 @@
 
-### How to Build OHS domain Image
-Go to folder **samples/1221-OHS-domain**
+        $ cd ../OracleJava/java-8
+        $ sh build.sh
 
-Run the following command:
+### Building OHS Docker Image
+IMPORTANT: You have to download the OHS binary and put it in place (see .download files inside dockerfiles/).
 
-        $ docker build --force-rm=true --no-cache=true --rm=true -t sampleohs:12.2.1 .
+Download the required package (see .download file) and drop them in the folder 12.2.1.2.0. Then go into the **dockerfiles** folder and run the **buildDockerImage.sh** script as root.
 
-Verify you now have this image in place with
+    $ sh buildDockerImage.sh -v 12.2.1.2.0
 
-        $ docker images
+IMPORTANT: The resulting image will have a  pre-configured domain. 
 
 ### How to run container
 
-**Prerequisite** : Create a docker data volume which will contain the Oracle Weblogic Proxy Plugin file
+If we want to start the OHS container without specifying any configuration for mod_weblogic:
+1. To start the OHS container with above oracle/ohs:12.2.1.2.0-sa image, run the following command:
 
-         Eg:$ docker volume create --name volume
+         docker run -it --name ohs -p 7777:7777 oracle/ohs:12.2.1.2.0-sa
 
-_This volume will be created in "/var/lib/docker" directory or the location where "/var/lib/docker" points to._
 
+If we want to start the OHS container with some pre-specified mod_weblogic configuration:
+1. Depending on your weblogic environment , create a **custom_mod_wl_ohs.conf** file by referring to container-scripts/mod_wl_ohs.conf.sample and section 2.4 @ [OHS 12c Documentation](http://docs.oracle.com/middleware/12212/webtier/develop-plugin/oracle.htm#PLGWL553)
 
-1. Depending on your weblogic environment , create a **custom_mod_wl_ohs.conf** file by referring to container-scripts/mod_wl_ohs.conf.sample and section 2.4 @ [OHS 12c Documentation](http://docs.oracle.com/middleware/1221/webtier/develop-plugin/oracle.htm#PLGWL553)
+2. Place the custom_mod_wl_ohs.conf file in a directory in the host say,"/scratch/DockerVoulme/OHSVolume" and then mount this directory into the container as a volume.
 
-2. Place the custom_mod_wl_ohs.conf file in docker data volume directory . e.g /var/lib/docker/volume
-
-3. To start the OHS Container with above sampleohs:12.2.1 image , run command from docker voume directory
+3. To start the OHS Container with above oracle/ohs:12.2.1.2.0-sa image , run command from the directory which ha sbeen mounted as a volume.
 
          For e.g
-         $ cd /var/lib/docker/volume
-         $ docker run -v `pwd`:/volume -w /volume -d --name ohs -p 7777:7777  sampleohs:12.2.1 configureWLSProxyPlugin.sh
+         $ cd /scratch/DockerVolume/OHSVolume
+         $ docker run -v `pwd`:/volume -w /volume -d --name ohs -p 7777:7777  oracle/ohs:12.2.1.2.0-sa configureWLSProxyPlugin.sh
```

Comment:
You shouldn't need to specify `configureWLSProxyPlugin.sh`. Rather, your `CMD` script should automatically detect if a custom configuration has been provided and fire the script itself.